config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-periodics-config.yaml

```diff
@@ -0,0 +1,890 @@
+periodics:
+  kubernetes-sigs/azuredisk-csi-driver:
+  - name: periodic-24h-azuredisk-csi-driver-verify
```

Comment:
this is for driver verification test, not necessary in this periodic test.

Reply:
removed.